proxies/openfeature-nodejs/package.json

```diff
@@ -1,8 +1,8 @@
 {
   "dependencies": {
     "@devcycle/openfeature-nodejs-provider": "*",
-    "@openfeature/core": "^0.0.22",
-    "@openfeature/server-sdk": "^1.9.0",
+    "@openfeature/core": "*",
```

Comment:
putting * here doesn't change anything, it will just be locked to whatever version it first resolves to

Reply:
that's fine, but the intention with these is to use the latest, which I think setting as `*` implies.

Reply:
it doesn't really, I think it falsely gives the impression that it will always use latest when it actually won't